geode-assembly/src/acceptanceTest/resources/org/apache/geode/client/sni/geode-config/gemfire.properties

```diff
@@ -15,4 +15,5 @@
 # limitations under the License.
 #
 
-#empty
+statistic-sampling-enabled=true
```

Comment:
this turns on stats for all our SNI tests—not just this one. I assume that is ok but just wanted to remind everybody

Reply:
yes, it does.  I think that's okay.